README.md

```diff
@@ -69,7 +69,8 @@ imgpkg push -b ${REPO_HOST}:${TAG} -f repository
 
 The tanzu-application-platform-reference-service-packages project team welcomes contributions from the community. Before you start working with this project please
```

Comment:
*nitpick* We should update the README table to include the new elasticcache pacakge